go/tasks/plugins/webapi/bigquery/plugin.go

```diff
@@ -456,7 +457,7 @@ func (p Plugin) newBigQueryClient(ctx context.Context, identity google.Identity)
 		options = append(options,
 			option.WithEndpoint(p.cfg.bigQueryEndpoint),
 			option.WithTokenSource(oauth2.StaticTokenSource(&oauth2.Token{})))
-	} else {
+	} else if p.cfg.GoogleTokenSource.Type != "default" {
```

Comment:
is the else part use no token source? Can we log that?

Reply:
yeah.
BQ client will read env `GOOGLE_APPLICATION_CREDENTIALS` by default